### DISCUSSION TITLE
Help on integrating with nvim-cmp and copilot.vim

### DISCUSSION THREAD

**Discussion by Sirikakire** [CopilotC-Nvim/CopilotChat.nvim#316] [Q&A]
A simple wonder, Im trying to interact with nvim-cmp and copilot.vim suggestion in copilot chat window but couldn't:

- nvim-cmp: I config my keybinding on cmp so when I hit <cr>, it will choose the selected item and allow me to continue to write....But when hitting <cr> in copilot chat, it choose the selected item and also submit the prompt right after that.

- copilot.vim: Hitting <tab> should select the suggestion from copilot.vim but right now in copilot chat it didnt do any action

![image](https://github.com/CopilotC-Nvim/CopilotChat.nvim/assets/103923561/8905ed35-1f7d-40a9-966b-2f26c31f5e37)

**Comment by tecfu** [comment]
Why was https://github.com/CopilotC-Nvim/CopilotChat.nvim/issues/315 locked and closed? 

---

I am also seeing this issue. When hitting the tab key for completion in the chat window I get no result. Also using nvim-cmp.
